### ISSUE TITLE
Launch from command line support

### ISSUE THREAD

**Issue by ignlg** [dvcrn/markright#22]
To be able to launch it from command line is a step forward for all devs out there.

So we could use it like:

```shell
$ alias markright="~/Applications/MarkRight.app/Contents/SharedSupport/bin/markright"
$ markright README.md
```

It's beautiful, isn't it? (♥‿♥)

**Comment by derhuerst** [comment]
:+1:

**Comment by dvcrn** [comment]
I think this should be fairly simple once we have 'open with' support https://github.com/dvcrn/markright/issues/6. Seems to be straight forward - https://github.com/atom/electron/blob/master/docs/api/app.md#event-open-file. I'll see if I can get something running by the end of the day

**Comment by dvcrn** [comment]
All right, once #6 is merged we can do something like 

```
alias markright="open -a /Applications/MarkRight.app"
markright "FOO.md"
```

**Comment by dvcrn** [comment]
Looks like this is not completely done yet. When the app is closed, the command above will open the app but not load the document in. Only when the app is already running it will work.

**Comment by dvcrn** [comment]
Resolved with the above command in [0.1.10](https://github.com/dvcrn/markright/releases/tag/0.1.10)
